ecobee/objects.go

```diff
@@ -171,11 +171,17 @@ type Runtime struct {
 	RuntimeInterval    int    `json:"runtimeInterval"`
 	ActualTemperature  int    `json:"actualTemperature"`
 	ActualHumidity     int    `json:"actualHumidity"`
+	RawTemperature     int    `json:"rawTemperature"`
+	ShowIconMode       int    `json:"showIconMode"`
 	DesiredHeat        int    `json:"desiredHeat"`
 	DesiredCool        int    `json:"desiredCool"`
 	DesiredHumidity    int    `json:"desiredHumidity"`
 	DesiredDehumidity  int    `json:"desiredDehumidity"`
 	DesiredFanMode     string `json:"desiredFanMode"`
+	ActualVOC          int    `json:"actualVOC"`
+	ActualCO2          int    `json:"actualCO2"`
+	ActualAQAccuracy   int    `json:"actualAQAccuracy"`
+	ActualAQScore      int    `json:"actualAQScore"`
```

Comment:
These fields are not documented at https://www.ecobee.com/home/developer/api/documentation/v1/objects/Runtime.shtml, but seem reasonable for the newer models.  (The API documentation doesn't seem to have been updated in a few years.)